doc/bisect.rst

```diff
@@ -0,0 +1,87 @@
+*************************
+Code coverage with bisect
+*************************
+
+Normally, preprocessors (specified in ``dune`` files) are constant across all
+builds. However, it usually doesn't make sense to analyze code coverage on all
+builds, especially if you aren't running tests every time.
+
+In this section, we will explain how to set up code coverage with bisect_ppx_ so
+that you can enable and disable coverage via ``dune-workspace`` files.
+
+Specifying what to bisect
+=========================
+
+To measure coverage for every build, we can include ``bisect_ppx`` in the
+``preprocess`` field for each relevant library and executable. For example, the
+dune file would include stanza(s) like this:
+
+.. code:: scheme
+
+          (library
+           (name foo)
+           (preprocess (pps bisect_ppx)))
+
+This aligns with the standard usage of ppx libraries.
```

Comment:
I would drop this part as well and present directly the "right way" of using bisect_ppx with dune. Or alternatively put it as a note after the general usage.

If find that for this sort of documentation, it is better to answer the general usage questions first and leave things that are more implementation details for people who want to go further and understand how things work. For instance, here we could just mention that  `(bisect_ppx)` is the same as `(preprocess (pps bisect_ppx))` when bisect is enabled and is ignored otherwise.